tests/e2etest-xstoragebucket-kcl-1-18/kcl.mod.lock

```diff
@@ -0,0 +1,12 @@
+[dependencies]
+  [dependencies.model]
+    name = "model"
+    full_name = "models_0.0.1"
+    version = "0.0.1"
+    reg = "ghcr.io"
+    repo = "kcl-lang/model"
+    oci_tag = "0.0.1"
```

Comment:
Is this dep intentional? Doesn't look like it's used.

Reply:
this gets added by the new kcl-cli 
i wonder if we simple add kcl.mod.lock to ignore ?